doc/src/special_topics/finite_diff_derivatives.rst

```diff
@@ -249,5 +249,6 @@ given above.
 Next we show how to perform these and many other discritizations of derivatives,  but using a
 much more efficient approach originally due to Bengt Fornberg and now incorporated into SymPy.
 
+:doc:`/src/tutorial/calculus`
```

Comment:
can this be made to take one directly to `https://github.com/sympy/sympy/blob/master/doc/src/tutorial/calculus.rst#finite-differences`?

Reply:
Yes, I could do that